Development environment/Docker.md

```diff
@@ -1,16 +1,137 @@
 In order to install a WordPress development environment on [Docker](https://www.docker.com/), you'll need to install Docker and use [docker-compose](https://docs.docker.com/compose/).
```

Comment:
This entire file shouldn't be touched. Maybe some things like `DevOps` or small typos. But the suggestion is an old file.